lib/src/scss-shared/_nx-variables.scss

```diff
@@ -58,3 +58,6 @@ $nx-form-element-width-wide: 624px;
 $nx-tile-scroll-container-z-index: 0;
 $nx-submit-mask-z-index: 10;
 $nx-modal-backdrop-z-index: 1050;
+
+// table cell header
+$nx-table-celle-header-height: 48px;
```

Comment:
Since this variable is specific to the tables it can be defined in `_nx-tables.scss`

Reply:
Done: https://github.com/sonatype/sonatype-react-shared-components/pull/165/commits/ee72b488d3e7a87119e0736c14c050e0b39c3250